spec.md

```diff
@@ -215,6 +215,30 @@ help intermediate gateways determine how to route the events.
     * urn:event:from:myapi/resourse/123
     * mailto:cncf-wg-serverless@lists.cncf.io
 
+### subject
+* Type: `String`
```

Comment:
Perhaps Subject is a URIRef? and it relates to the source uri?

Reply:
It's some sort of subject identifier. I can be a filename or just some name of a thing that the source knows. I wouldn't constrain that as much as source.

Reply:
URIRef does all:

`this is a valid uri ref`

`http://so.is/this`